### PR TITLE
rock 5c: compatiable with more overlays

### PR DESCRIPTION
**Pull request by vamrs-feng**
compatiable with 5a overlays

### CODE REVIEW

File: arch/arm64/boot/dts/rockchip/overlays/rk3588-can1-m1.dts [hunk 1]
```diff
@@ -6,7 +6,7 @@
 / {
 	metadata {
 		title = "Enable CAN1-M1";
-		compatible = "radxa,rock-5a", "radxa,rock-5b", "radxa,rock-5b-plus";
+		compatible = "radxa,rock-5a", "radxa,rock-5b", "radxa,rock-5c", "radxa,rock-5b-plus";
```
Comment (RadxaYuntian) [inline arch/arm64/boot/dts/rockchip/overlays/rk3588-can1-m1.dts:9]: 字母顺序要放在5b+后面

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

**Comment by RadxaYuntian** [comment]
你修改好了点一下re-request review
image

**Comment by vamrs-feng** [comment]
好，你先review一下，可以了我在合并成一个提交吧？

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
squash吧

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**